### DISCUSSION TITLE
SiamMask Train

### DISCUSSION THREAD

**Discussion by jianjiandandande** [STVIR/pysot#505] [Q&A]
https://github.com/STVIR/pysot/blob/2f98bfb6f7b56e4a4263c7024a3a3a7143d630ac/pysot/models/model_builder.py#L112

I find the mask_loss is not initialized

**Comment by EEEEEREN** [comment]
这里应该设置多少呢...

**Comment by jianjiandandande** [comment]
> 这里应该设置多少呢...
他这个项目里面没有对SiamMask做具体的实现，只是搭了部分模块出来。这里的loss压根没有定义出来，建议参考王强大佬的原SiamMask去做复现

**Comment by EEEEEREN** [comment]
> > 这里应该设置多少呢...
> > 他这个项目里面没有对SiamMask做具体的实现，只是搭了部分模块出来。这里的loss压根没有定义出来，建议参考王强大佬的原SiamMask去做复现

我感觉也是，在这里搞SiamMask有点难，遇到了好多错误就是训不出，连训练界面都出不来😂
我在王强大佬自己的Siammask里面已经复现了，我看pysot的东西挺多的，就也打算在这里尝试复现
